unfoldingWord/translationCore
# Removing Requirements.js

#### This pull request addresses:
Cleaning up code base #1773

#### How to test this pull request:
Make sure everything works as expected. QA pre-flight


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1786)
